less/navigation.less

```diff
@@ -1 +1,18 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.logo-heading {
+    font-size: 4rem;
+    .center();
+    padding: 10px;
+}
+
+nav {
```

Comment:
I didn't get to see it because it was so subtle, don't forget to look at the design file. There is supposed to be a background on the nav